content/community/gsoc-2022.de.md

```diff
@@ -34,6 +34,8 @@ Schauen dir auch unsere Medienkanäle an!
 - [Mastodon/Fediverse](https://fosstodon.org/@xmpp/)
 - [Youtube](https://www.youtube.com/c/XMPPStandardsFoundation)
 
+Darüber hinaus seid ihr eingeladen, mit unserem [Flyer vor Ort zu werben](https://github.com/xsf/xmpp.org/blob/master/static/images/promo/Flyer_XMPP_GSoC2022_DE.pdf), wie z.B. ein schwarzes Brett, für die bevorstehende Veranstaltung. Bitte seid so nett und fragt, ob ihr den Flyer aufhängen dürft.
```

Comment:
The link would have to be `/images/promo/Flyer_XMPP_GSoC2022_DE.pdf`

Same goes for the EN variant

Reply:
Shiat...